### PR TITLE
Added Create migration environment and delete migration environment

### PR DESCRIPTION
**Pull request by rawstorage**
functions to fully cover NDM setup and tear down.  Tests have been
updated.

### CODE REVIEW

File: PyU4V/migration.py [hunk 1]
```diff
@@ -53,6 +53,29 @@ def get_migration_info(self):
             version=self.U4V_VERSION, array_id=self.array_id)
         return self.common.get_request(target_uri, 'migration info')
 
+    def create_migration_environment(self, target_array_id):
+        """Creates a new migration environment between two arrays for use
+        with non disruptive migrations
+
+        :param target_array:
+        :return: dict
```
Comment (helenwalsh) [inline PyU4V/migration.py]: :returns:

File: PyU4V/migration.py [hunk 2]
```diff
@@ -53,6 +53,29 @@ def get_migration_info(self):
             version=self.U4V_VERSION, array_id=self.array_id)
         return self.common.get_request(target_uri, 'migration info')
 
+    def create_migration_environment(self, target_array_id):
+        """Creates a new migration environment between two arrays for use
+        with non disruptive migrations
+
+        :param target_array:
```
Comment (odonos12) [inline PyU4V/migration.py]: :param target_array: should be target_array_id, please add a description like you have in the delete_migration_environment method too.

Comment (odonos12) [reply PyU4V/migration.py]: Changes implemented in commit 5426cb4 - resolving.

**Review by odonos12 — APPROVED** [review]
Looks good.

File: PyU4V/migration.py (hunk 1 above)
Comment (rawstorage) [reply PyU4V/migration.py]: Made the requested change

**Review by helenwalsh — APPROVED** [review]

File: PyU4V/migration.py [hunk 3]
```diff
@@ -57,8 +57,8 @@ def create_migration_environment(self, target_array_id):
         """Creates a new migration environment between two arrays for use
         with non disruptive migrations
 
-        :param target_array:
-        :return: dict
+        :param target_array: 12 Digit serial number of target array
```
Comment (odonos12) [inline PyU4V/migration.py]: :param target_array: should be :param target_array**_id**:

**Pull request merged**